### DISCUSSION TITLE
"Disabled people" should be in the list of accepted terms

### DISCUSSION THREAD

**Discussion by NickColley** [retextjs/retext#81] [General]
While in the US there are a lot of people that prefer "People with disabilities (PWD)" the UK and increasingly more generally "Disabled people" is the preferred identifier.
This often depends on where a disabled person aligns politically, for me I use 'disabled person' because I am 'identity first' I think my disabilities are an important part of who I am.
I don't like "Person with a disability" because it can lead to trying to avoid or erase it, however some people really like it because they dont like to be defined exclusively by their disability.

There is no 'right' version they're both good, so we should amend the logic to be like:
"The Disabled, Disableds, Invalids, [insert obvious slurs here]" -> Bad
"Disabled people" -> Good
"People with disabilities" -> Good

**Comment by wooorm** [comment]
Hi, thanks for raising this!

I’d like that add:
a) it depends on the disability or identity, for example the autistic, the blind as groups largely prefer those terms, whereas people with AIDS largely prefer that term
b) it’s changing over time, the consensus was more clearly about person-first language when this project was created, and is moving towards identity-first
c) there is still a difference between how groups talk about themselves, or a friend of that group, and how say for example news organizations, scientific papers, or governments should talk about these folks

I can think of several solutions to this:
- Let users choose whether they’d want to allow one or the other
- Allow both person-first and identity-first language as acceptable
- Build a rather complex system on which group prefers what, in which regions

The current state is somewhat the third point, groups have been changing terms they identify with in this project. However, I think it’s so complex with regions and different contexts, that it’s impossible to get right.

**Comment by NickColley** [comment]
I think the most pragmatic way forward is to make the logic looser and focus on what is agreed by majority of people to be bad rather than attempting contextual changes.

**Reply by wooorm** [reply]
Sure. Is this something you are interested in working on?

**Reply by NickColley** [reply]
Yeah sounds good, I can't promise anything super timely though sorry.
